### PR TITLE
[kube-proxy]: Remove unused util functions

### PR DESCRIPTION
**Pull request by HirazawaUi**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
They are no longer used.

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
Just saw the useless function while reading the code, feel free to close this PR if I'm doing something wrong.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/proxy/util/utils.go [hunk 1]
```diff
@@ -261,20 +194,6 @@ func AppendPortIfNeeded(addr string, port int32) string {
 	return fmt.Sprintf("[%s]:%d", addr, port)
 }
 
-// ShuffleStrings copies strings from the specified slice into a copy in random
-// order. It returns a new slice.
-func ShuffleStrings(s []string) []string {
```
Comment (danwinship) [inline pkg/proxy/util/utils.go:266]: unused after the removal of the userspace proxy in https://github.com/kubernetes/kubernetes/pull/112133

File: pkg/proxy/util/utils.go [hunk 2]
```diff
@@ -313,18 +232,6 @@ func GetClusterIPByFamily(ipFamily v1.IPFamily, service *v1.Service) string {
 	return ""
 }
 
-// RevertPorts is closing ports in replacementPortsMap but not in originalPortsMap. In other words, it only
-// closes the ports opened in this sync.
-func RevertPorts(replacementPortsMap, originalPortsMap map[netutils.LocalPort]netutils.Closeable) {
```
Comment (danwinship) [inline pkg/proxy/util/utils.go:318]: unused after removal of the "port opener" code in https://github.com/kubernetes/kubernetes/pull/108496

**Review by danwinship — APPROVED** [review]
Yup, looks like this is all dead code.
/lgtm
/approve

(nitpick if you do any similar PRs in the future: I'd call them "unused" functions, not "useless")

**Comment by HirazawaUi** [comment]
> (nitpick if you do any similar PRs in the future: I'd call them "unused" functions, not "useless")

All right and thank you very much for tracing to when these codes are not used

**Pull request merged**